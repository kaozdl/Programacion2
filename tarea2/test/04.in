insertar_lista (16,marzo) (27,abril) (13,abril) .
modificar_lista 0 13
modificar_lista 27 27
modificar_lista 27 16
modificar_lista 13 27
modificar_lista 13 16
Fin
