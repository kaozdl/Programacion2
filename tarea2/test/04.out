1>(0,primero)(16,marzo)(27,abril)(13,abril)
2>(13,abril)(16,marzo)(27,abril)(0,primero)
(0,primero)(13,abril)(16,marzo)(27,abril)
3>(0,primero)(13,abril)(16,marzo)(27,abril)
(0,primero)(13,abril)(16,marzo)(27,abril)
4>(0,primero)(13,abril)(27,abril)(16,marzo)
(0,primero)(13,abril)(27,abril)(16,marzo)
5>(0,primero)(27,abril)(13,abril)(16,marzo)
(0,primero)(13,abril)(27,abril)(16,marzo)
6>(0,primero)(16,marzo)(27,abril)(13,abril)
(0,primero)(13,abril)(16,marzo)(27,abril)
7>Fin.
